plaso/cli/status_view.py

```diff
@@ -385,14 +389,17 @@ def SetMode(self, mode):
     """
     self._mode = mode
 
-  def SetSourceInformation(self, source_path, source_type, filter_file=None):
+  def SetSourceInformation(self, source_path, source_type,
```

Comment:
I opt to keep the style consistent with the rest of the source, so please line break after (

Reply:
Done